.prettierignore

```diff
@@ -0,0 +1,5 @@
+# Add files here to ignore them from prettier formatting
+
+/dist
+/coverage
+
```

Comment:
I think it is better if we leave it here as it might happen that somebody runs `npm` or some other 3rd party task does it. Better to be safe than sorry.